src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -65,9 +66,15 @@ public void execute_commandExecutionError_throwsCommandException() {
     }
 
     @Test
-    public void execute_validCommand_success() throws Exception {
-        String listCommand = ListEmployeeCommand.COMMAND_WORD;
-        assertCommandSuccess(listCommand, ListEmployeeCommand.MESSAGE_SUCCESS, model);
+    public void execute_validListEmployeesCommand_success() throws Exception {
+        String listEmployeeCommand = ListEmployeeCommand.COMMAND_WORD;
+        assertCommandSuccess(listEmployeeCommand, ListEmployeeCommand.MESSAGE_SUCCESS, model);
+    }
+
+    @Test
+    public void execute_validListProjectsCommand_success() throws Exception {
```

Comment:
This should be `execute_validListProjectCommand_success`, i.e. Project without the 's' at the end, since the name of the command itself is `ListProjectCommand`.